README.md

```diff
@@ -9,11 +9,35 @@ See https://wiki.debian.org/RaspberryPi3#Preview_image for where to obtain the l
 
 ## Option 2: Building your own image
 
-If you prefer, you can build a Debian buster Raspberry Pi 3 image using:
+If you prefer, you can build a Debian buster Raspberry Pi 3 image yourself. For
+this, first install the
+[requirements](https://github.com/larswirzenius/vmdb2/blob/master/README#getting-vmdb2)
+of vmdb2. Then run the following:
 
 ```shell
 git clone --recursive https://github.com/Debian/raspi3-image-spec
 cd raspi3-image-spec
+```
+
+Then edit [raspi3.yaml](raspi3.yaml) to select the Debian repository that you
+want to use:
+
+- If you want to use the snapshot with which the build was tested, use
+    `http://snapshot.debian.org/archive/debian/20171007T213914Z/`. This is what
+    is pre-configured in raspi3.yaml. However, due to a [missing
+    feature](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=763419) on
+    snapshots, to make the build work, you have to disable an expiration check
+    by APT. To do so, edit raspi3.yaml to replace all
+    `apt-get` invocations with `apt-get -o Acquire::Check-Valid-Until=false`
+- If you want to use the latest versions of each software, you can replace
+    `http://snapshot.debian.org/archive/debian/20171007T213914Z/` in raspi3.yaml
+    with the URL of your favorite Debian mirror. Of course, this means that the
```

Comment:
s,the URL of your favorite Debian mirror,http://deb.debian.org/debian, please. People who care about mirrors will know how to replace it.

Reply:
OK great idea! fixing